buildkite/scripts/rosetta-integration-tests.sh

```diff
@@ -222,100 +176,56 @@ until [ $daemon_status == "Synced" ]; do
 echo "==================== UNLOCKING GENESIS ACCOUNTS ======================"
 mina-dev accounts unlock --public-key $BLOCK_PRODUCER_PK
 mina-dev accounts unlock --public-key $SNARK_PRODUCER_PK
 
-# Start sending payments
-send_payments() {
+# Start sending value transfer transactions
+send_value_transfer_txns() {
   mina-dev client send-payment -rest-server http://127.0.0.1:${MINA_GRAPHQL_PORT}/graphql -amount 1 -nonce 0 -receiver $BLOCK_PRODUCER_PK -sender $BLOCK_PRODUCER_PK
   while true; do
     sleep $TRANSACTION_FREQUENCY
     mina-dev client send-payment -rest-server http://127.0.0.1:${MINA_GRAPHQL_PORT}/graphql -amount 1 -receiver $BLOCK_PRODUCER_PK -sender $BLOCK_PRODUCER_PK
   done
 }
-send_payments &
-
-# Fee payer cache creation
-echo "==================== PREPARE FEE PAYER CACHE ======================"
-zkapp_fee_payer_pk=$(cat ${MINA_KEYS_PATH}/zkapp-${zkapp}-fee-payer.key.pub)
-zkapp_fee_payer_privkey=$(mina-dev advanced dump-keypair --privkey-path "${MINA_KEYS_PATH}/zkapp-${zkapp}-fee-payer.key" | sed -ne "s/Private key: //p")
-
-mkdir -p /root/.cache/zkapp-cli/keys
-echo -e "{\n    \"privateKey\": \"${zkapp_fee_payer_privkey}\",\n    \"publicKey\": \"${zkapp_fee_payer_pk}\"\n}" >/root/.cache/zkapp-cli/keys/sandbox.json
-
-# Deploy zkApps
-echo "==================== DEPLOYING ZKAPPS ======================"
-echo "If this fails, it's likely due to incompatibility between the
-o1js and zkapp-cli versions in use."
-echo "NOTE: At the moment the daemon still has an old version of
-      snarkyjs pinned to it, so we cannot use the latest version of
-      zkapp-cli, which requires o1js. Once the pinned snarkyjs version
-      gets updated, this build will most likely fail and we will then
-      need to update the zkapp-cli version used here. This is
-      unfortunate, but necessary. Please delete this warning once it's
-      done." > /dev/stderr
-
-deploy_txs=()
-for zkapp_path in ${ZKAPP_PATH}/*/; do
-  zkapp_path=${zkapp_path%/}
-  zkapp=$(basename $zkapp_path)
-  echo "Deploying ${zkapp}..."
-
-  zkapp_account_pk=$(cat ${MINA_KEYS_PATH}/zkapp-${zkapp}-account.key.pub)
-  zkapp_account_privkey=$(mina-dev advanced dump-keypair --privkey-path "${MINA_KEYS_PATH}/zkapp-${zkapp}-account.key" | sed -ne "s/Private key: //p")
-
-  mkdir -p ${zkapp_path}/keys
-  echo -e "{\n    \"privateKey\": \"${zkapp_account_privkey}\",\n    \"publicKey\": \"${zkapp_account_pk}\"\n}" >"${zkapp_path}/keys/sandbox.json"
-
-  cat <<EOF >"${zkapp_path}/config.json"
-{
-  "version": 1,
-  "networks": {
-    "sandbox": {
-      "url": "http://127.0.0.1:${MINA_GRAPHQL_PORT}/graphql",
-      "keyPath": "keys/sandbox.json",
-      "feepayerKeyPath": "/root/.cache/zkapp-cli/keys/sandbox.json",
-      "feepayerAlias": "sandbox",
-      "fee": "1"
-    }
-  }
-}
-EOF
-  cd "$zkapp_path"
-  npm ci
-  npm run build
-  txn=$(zk deploy sandbox -y | sed -ne "s/https:\/\/berkeley.minaexplorer.com\/transaction\///p")
-  deploy_txs+=txn
-  cd -
-  echo "Done."
-done
```

Comment:
I see the `send_zkapp_transactions` mechanism replaces it, and if it creates balance-changing operations, then I think it will suffice for these tests and it's a lot simpler than the previous zkapp deploy/interact mechanism. What's your opinion on this?

Reply:
Well, I agree that including `o1js` and `zkapp-cli` just to test Rosetta complicates things more than should be necessary. If `send_zkapp_transactions` replaces it reasonably to generate money transfers, then that's fine. But I'm not sure if that's the case at the moment.

Reply:
In Rosetta we have two types of transactions (`Zkapp_fee_payer_dec` and `Zkapp_balance_change`), so if it produces these two operations (which I believe it does) it should be enough. I will do an offline run to check this behaviour. Also, it seems that the script is failing on the CI run.

Reply:
I can confirm this error locally.

```
Command Failed: unable to sync to 4: unable to sync to 4: unable to fetch block 4: unable to fetch block 4: unable to fetch block 4 from network sandbox with retry: /block {"index":4} error {Code:1 Message:SQL failure Description:0xc04560f950 Retriable:false Details:map[body:[Sql Cannot decode int from <postgres://pguser:_@127.0.0.1:5432/archive>: Invalid value "f".] error:Cannot decode int from <postgres://pguser:_@127.0.0.1:5432/archive>: Invalid value "f". extra:Finding zkapp account updates within command]}: request failed
```

Reply:
I expect https://github.com/MinaProtocol/mina/pull/14224/commits/4702bc9f6bf52207118460e5f93c1f1d2ff2be3a to fix this issue.